changelog.md

```diff
@@ -3,7 +3,7 @@ This file contains all the notable changes done to the Ballerina Constraint pack
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [1.0.0] - 2022-09-08
```

Comment:
```suggestion
## [Unreleased]

### Changed

API docs updated

## [1.0.0] - 2022-09-08
```

Reply:
Fixed in [f5f939c](https://github.com/ballerina-platform/module-ballerina-constraint/pull/27/commits/f5f939c7785603a0c1f388fd75f4792df97bdb38)